### DISCUSSION TITLE
Fixed wing Aircraft in Airsim

### DISCUSSION THREAD

**Discussion by redhat2299** [microsoft/AirSim#4194] [Support Q&A]
Can anyone help me in integration of fixed wing aircraft in Airsim Please help out  @rajat2004 @AOS55

**Comment by zimmy87** [comment]
We were able to run @AOS55's Fixed Wing module at https://github.com/AOS55/Fixedwing-Airsim using his readme.md. There's currently support for the x8 skywalker drone and the cessna flight model in JSBSim. More work will probably be required to support other aircraft.

**Comment by zclyaya** [comment]
> Can anyone help me in integration of fixed wing aircraft in Airsim Please help out @rajat2004 @AOS55

hello,  Do you intergrate it? I meet some problem,can you help me ?
